modules/blazingsql/wrapper/blazingsql/contextwrapper.cpp

```diff
@@ -13,25 +13,55 @@
 // limitations under the License.
 
 #include "contextwrapper.hpp"
+#include <execution_graph/Context.h>
+#include <node_cudf/table.hpp>
+#include "cache.hpp"
 
 namespace nv {
 
-Napi::Function ContextWrapper::Init(Napi::Env env, Napi::Object exports) {
+Napi::Function ContextWrapper::Init(Napi::Env const& env, Napi::Object exports) {
   return DefineClass(env, "ContextWrapper", {});
 }
 
 ContextWrapper::wrapper_t ContextWrapper::New(
   Napi::Env const& env,
+  int32_t const& ral_id,
   std::pair<
     std::pair<std::shared_ptr<ral::cache::CacheMachine>, std::shared_ptr<ral::cache::CacheMachine>>,
-    int> args) {
-  auto inst    = EnvLocalObjectWrap<ContextWrapper>::New(env, {});
-  auto& caches = args.first;
-  inst->_port  = args.second;
+    int> const& pair,
+  Napi::Object const& ucp_context) {
```

Comment:
We can strongly-type this argument:

```suggestion
  UcpContext::wrapper_t const& ucp_context) {
```